src/Sidekick.Protocol/Program.cs

```diff
@@ -1,14 +1,25 @@
 using System;
```

Comment:
I feel like the protocol should be more unique, instead of `sidekick:`, something like `sidekickpoe:`.
I remember a browser named "Sidekick" had the same install dir as sidekick.

Reply:
Created an issue specific to this change